### DISCUSSION TITLE
Sign in failed. Reason: Could not log in with device flow on http: HTTP 504, request id: 19, error code: 1001

### DISCUSSION THREAD

**Discussion by krishnadesai0987** [community/community#128936] [Copilot Conversations]
### Select Topic Area

Bug

### Body

i am unable to login to github copilot , 

failing with error :  Sign in failed. Reason: Could not log in with device flow on http: HTTP 504, request id: 19, error code: 1001

i was able to login earlier.

please help me with the solution

**Comment by Akash1134** [comment]
Hi @krishnadesai0987 ,

This is most likely an issue with your network. I'd recommend checking our [network troubleshooting guide](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot).

**Comment by blueair5** [comment]
In recent days, I also meet this problem in IDEA. Because I use a proxy in my Mac. My solution is follow：
1. when you login in github copilot, set IDEA proxy. (https://docs.github.com/en/copilot/managing-copilot/configure-personal-settings/configuring-network-settings-for-github-copilot)
2. After I successfully logged into copilot， I cancel IDEA proxy, because use proxy, copilot's answer is very slow.
